nanover-rs/src/tracked_simulation/openmm.rs

```diff
@@ -161,12 +161,21 @@ impl TrackedSimulation for TrackedOpenMMSimulation {
     ) -> Result<(), BroadcastSendError> {
         let mut frame = self.simulation.to_framedata(with_velocities, with_forces);
         let energy_total = self.user_energies();
+        let potential_energy = self.simulation.get_potential_energy();
```

Comment:
This function involves accessing the OpenMM state, which obviously has to happen in .to_framedata() anyway. Ideally we would not need to get the potential energy again, but just amend the value set elsewhere by the correction calculated in this file, removing the necessity for this function. I will add a comment to make this clearer

Reply:
I have added a comment and a TODO addressing this